pkg/kubelet/types/types.go

```diff
@@ -68,6 +68,21 @@ func (s SortedContainerStatuses) Less(i, j int) bool {
 	return s[i].Name < s[j].Name
 }
 
+// Sorts InitContainerStatuses
```

Comment:
Minor thing, but we try to stay consistent to Godoc style which is `SortInitContainerStatuses ensures that statuses are in the order that their init container appears in the pod spec."  I.e. one sentence minimum, starts with the name of the function.